src/libs/NumberFormatUtils.ts

```diff
@@ -1,12 +1,25 @@
+import stableStringify from 'fast-json-stable-stringify';
```

Comment:
Can you explain what this library is for like I'm 5?

Reply:
It is like json stringify but with ordered keys. So if we pass an option object with keys in different orders it will return the same string.

For example:

```js
stringifyStable({a: "a", b: "b"}) == '{"a": "a", "b": "b"}'
stringifyStable({b: "b", a: "a"}) == '{"a": "a", "b": "b"}'
```

So in this case with NumberFormat option objects it means that it will use that same cached instance even if we pass the options in different order.

I also noticed this library was already included as a transitive dependency of some other library so we were already using it indirectly.

Reply:
thanks, created https://github.com/Expensify/App/issues/40190

Reply:
Thanks!

Reply:
@janicduplessis can you please try just replacing `stringifyStable` with `JSON.stringify` and see if it makes any difference in the trace?